Mathlib/Order/Partition/Finpartition.lean

```diff
@@ -666,7 +662,7 @@ theorem card_filter_atomise_le_two_pow (ht : t ∈ F) :
   rw [subset_iff]
   simp_rw [mem_image, mem_powerset, mem_filter, and_imp, Finset.Nonempty, exists_imp, mem_atomise,
     and_imp, Finset.Nonempty, exists_imp]
```

Comment:
Can you fix this simp call instead of changing the `rintro` invokation?

Reply:
Done.